scvi/model/_scanvi.py

```diff
@@ -366,6 +366,7 @@ def train(
             batch_size=batch_size,
             use_gpu=use_gpu,
         )
+        data_splitter.setup()
```

Comment:
Do we need to have these calls if setup() is already called in TrainRunner?

Reply:
It doesn't do anything after the first time it's called (is passed over), and it's more necessary to call here as it sets the `train_idx` attribute.